README.md

```diff
@@ -8,7 +8,8 @@ This project is a gateway between the Safe clients ([Android](https://github.com
 
 ## Documentation
 
-- [Client Gateway Wiki](https://safe.global/safe-client-gateway/)
+- [Client Gateway Docs](https://safe.global/safe-client-gateway/)
+- [Swagger](https://safe-client.gnosis.io/swagger/index.html)
```

Comment:
Maybe we should wait for #927 in production. What do you think?

Reply:
Yes makes sense! I was thinking on updating it afterwards but this is also not critical 👍